src/utils/facadeLocator.js

```diff
@@ -0,0 +1,23 @@
+function stripSlashes(path) {
+	return path.replace(/^(\/+)|(\/+)$/g, '');
+}
+
+const setupFacadeLocator = (app) => {
```

Comment:
`
function stripSlashes(path = '') {
	return path.replace(/^(\/+)|(\/+)$/g, '');
}
const facadeDict = {};
const setupFacadeLocator = (app) => {
app.facade = (path, facade) {
    const strippedPath = stripSlashes(path);
    if (facade) { // typeof === object , class or instanceof facade (as abstract class for extends this)
         facadeDict[strippedPath] = facade;
         return facade
    } else if(path !== '') {
         	return facadeDict[strippedPath];
    }
    return facadeDict
}`